test/unit/operations/list_collections.test.js

```diff
@@ -71,7 +145,22 @@ describe('ListCollectionsOperation', function () {
           listCollections: 1,
           cursor: {},
           filter: {},
-          nameOnly: false
+          nameOnly: false,
+          authorizedCollections: false
+        });
+      });
+    });
+
+    context('when authorizedCollections is not provided', function () {
```

Comment:
Similarly to my comment on the other test case, could we update that context in L140 to be `when no options are provided` and then either add the `it` block below to that same context, or even update the description in L143 to `it('sets nameOnly and authorizedCollections properties to false')`? Since we are using deep equality and not checking individual properties I think it would make sense to use a a test assertion that reflects the combined nature of the check.